service/integration/open_pbs/geopm_install_pbs_power_limit.sh

```diff
@@ -67,9 +108,15 @@ remove() {
 
 if [ $# -eq 0 ]; then
     install
+    # TODO: Print recommendation to restart the PBS server/scheduler? Or do it automatically?
```

Comment:
I'd vote to print the reccomendation.  The admin may want to wait till an opportune time to restart PBS.